protobufs/livekit_phone_number.proto

```diff
@@ -0,0 +1,129 @@
+  int32 limit = 1;                // Optional: Maximum number of results (default: 50)
+  TokenPagination page_token = 2;          // Optional: Token for pagination (empty for first page)
+}
+
+// ListPurchasedPhoneNumbersResponse - Response containing purchased phone numbers
+message ListPurchasedPhoneNumbersResponse {
+  repeated PurchasedPhoneNumber items = 1;  // List of purchased phone numbers
+  TokenPagination next_page_token = 2;               // Token for next page (empty if no more pages)
+  int32 total_count = 3;                    // Total number of purchased phone numbers
+}
+
+// ReleasePhoneNumberRequest - Request to release purchased phone numbers
+message ReleasePhoneNumberRequest {
+  repeated string phone_numbers = 1;  // Phone numbers to release (e.g., ["+1234567890", "+1234567891"])
+}
+
+// GlobalPhoneNumber represents a phone number with standardized format
+message GlobalPhoneNumber {
+  string id = 1;                  // unique identifier
+  string e164_format = 2;         // e.g., "+14155552671"
+  string country_code = 3;        // e.g., "US"
+  string area_code = 4;           // e.g., "415"
+  string number_type  = 5;        // e.g., local, toll-free, national, mobile
+  string locality = 6;            // City/locality (e.g., "San Francisco")
+  string region = 7;              // State/region (e.g., "CA")
+  double spam_score = 8;          // can be used later for fraud detection
+  google.protobuf.Timestamp created_at = 9;           // timestamp when created
+  google.protobuf.Timestamp updated_at = 10;          // timestamp when updated
+}
+
+// TelephonyCost represents the pricing structure for a specific telephony service
+message TelephonyCost {
+  TelephonyCostType resource = 1;       // Type of telephony service
+  string currency = 2;                  // Currency code (e.g., "USD", "EUR")
+  double price = 3;                     // Price for this service
+  string billing_unit = 4;              // Billing unit (e.g., "month", "minute", "message", "call")
+}
+
+
+
+// PhoneNumberInventoryItem - Represents an available phone number for purchase
+message PhoneNumberInventoryItem {
+  GlobalPhoneNumber phone_number = 1;     // Common phone number fields
+  repeated string capabilities = 2;       // Comma-separated capabilities (e.g., "voice,sms")
+  repeated TelephonyCost costs = 3;       // Array of costs for different services
+}
+
+// PurchasedPhoneNumber - Represents a phone number owned by a LiveKit project
+message PurchasedPhoneNumber {
+  GlobalPhoneNumber phone_number = 1;  // Common phone number fields
+  PhoneNumberStatus status = 2;        // Current status of the phone number
+  google.protobuf.Timestamp assigned_at = 3;               // Timestamp when the number was assigned
+  google.protobuf.Timestamp released_at = 4;               // Timestamp when the number was released (if applicable)
+  SIPDispatchRuleInfo sip_dispatch_rule = 5;  // Optional: Associated SIP dispatch rule
```

Comment:
should this be set on the dispatch rule itself? we already have the "inbound numbers" field in there